static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -32,18 +32,42 @@ import {
 
 export const getEpicTestToRun = memoize(
     (): Promise<?Runnable<EpicABTest>> => {
+        // The syncEpicTests are the hard-coded tests, and asyncEpicTests are fetched from config
```

Comment:
I think I've brought this up on a different PR, but can we change the names of the tests to reflect this information? 

`hardCodedTests` and `configuredTests` maybe?

Reply:
good point, have updated. If it needs a comment to explain the names then the names probably need to change!